code/modules/medical/computer/cloning.dm

```diff
@@ -16,6 +16,7 @@
 	var/datum/dna2/record/active_record = null
 	var/obj/item/weapon/disk/data/diskette = null //Mostly so the geneticist can steal everything.
 	var/loading = 0 // Nice loading text
+	var/available_species = list("human","tajaran","skrell","unathi","grey","plasmamen","vox")
```

Comment:
I was referring to your comment 'also vg doesnt even have most these races'

They've all existed on this server for a long time now in one form or another.